docs/Documentation/UserGuideV0.7.0/7-Tools-Sync.md

```diff
@@ -0,0 +1,278 @@
+<table>
+   <tr>
+      <td colspan="2">parameter: server_ip</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>IP address of sync receiver.</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>String</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>127.0.0.1</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart client</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: server_port</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>Listening port of sync receiver, it is necessary to ensure that the port is consistent with the configuration of the listening port set in receiver.</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>Short Int : [0,65535]</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>5555</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart client</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: sync_period_in_second</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>The period time of sync process, the time unit is second.</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>Int : [0,2147483647]</td>
+   </tr>
+   <tr>
+      <td>Default</td>
+      <td>600</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart client</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: iotdb_schema_directory</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>The absolute path of the sender's IoTDB schema file, such as $IOTDB_HOME/data/system/schema/mlog.txt (if the user does not manually set the path of schema metadata, the path is the default path of IoTDB engine). This parameter is not valid by default and is set manually when the user needs it.</td>
+   </tr>
+   <tr>
+      <td>Type</td>
+      <td>String</td>
+   </tr>
+   <tr>
+      <td>Modalities for Entry into Force after Modification</td>
+      <td>Restart client</td>
+   </tr>
+</table>
+
+<table>
+   <tr>
+      <td colspan="2">parameter: iotdb_bufferWrite_directory</td>
+   </tr>
+   <tr>
+      <td width="30%">Describtion</td>
+      <td>The absolute path of the buffer Write data (tsfile file) directory of the IoTDB at the sender to the buffer write directory, such as: $IOTDB_HOME/data/data/settled (if the user does not set the data path manually, the path is the default path of IoTDB engine). This parameter is not valid by default, and is set manually when the user needs it. This parameter needs to be guaranteed to belong to the same IoTDB as the parameter iotdb_schema_directory.</td>
```

Comment:
Can `"to the buffer write directory"` be deleted?

Reply:
Yes, fixed